packages/subscription-manager/src/components/SubscriptionManagerContainer/SubscriptionManagerContainer.tsx

```diff
@@ -12,18 +12,57 @@ export type SubscriptionManagerContainerProps = {
 	children?: React.ReactNode;
 };
 
+const getEmailAddress = () => {
+	const subkey: string | undefined = document.cookie
```

Comment:
Would you extract out the type of subkey here?

Reply:
Yes, indeed. One note: there's no need to type this variable, TypeScript can infer the type from the usage:

<img width="388" alt="CleanShot 2023-03-23 at 12 28 37@2x" src="https://user-images.githubusercontent.com/528287/227190423-36c91de7-5c7e-4c45-8d6b-cf26472ef40f.png">